### PR TITLE
chore(Styleguide): Controls prep

### PR DESCRIPTION
**Pull request by codecaaron**
## Overview
Wanted to start paving the way to Controls / Args as a replacement for knobs. 

The main way to do this is to make all stories a function of args.  This will make the adding them significantly easier and migrating from knobs easier down the line.

This is a large changeset, but should **NOT** result in a functional change (to current knobs or otherwise)

## Interpolation Prettiness 
Before
Screen Shot 2020-08-19 at 2 17 47 PM
After
Screen Shot 2020-08-19 at 2 17 28 PM

### PR Checklist

- [ ] Related to Abstract designs:
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.9.4-alpha.72e545.0<br/>@codecademy/gamut-styles@4.2.6-alpha.72e545.0<br/>@codecademy/gamut@12.3.4-alpha.72e545.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f35c0b8492e5d6d0c99f6bc--gamut-preview.netlify.app>https://5f35c0b8492e5d6d0c99f6bc--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f35c0b8492e5d6d0c99f6bc>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.1-alpha.b31a51.0<br/>@codecademy/gamut-styles@5.0.1-alpha.b31a51.0<br/>@codecademy/gamut@14.0.1-alpha.b31a51.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.1-alpha.198fbf.0<br/>@codecademy/gamut@14.0.1-alpha.198fbf.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.1-alpha.89dba7.0<br/>@codecademy/gamut@14.0.1-alpha.89dba7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f3d6aefbf67673c886dde19--gamut-preview.netlify.app>https://5f3d6aefbf67673c886dde19--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f3d6aefbf67673c886dde19>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.1-alpha.797d17.0<br/>@codecademy/gamut@14.0.1-alpha.797d17.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f3d6b09f49f611de773fb2b--gamut-preview.netlify.app>https://5f3d6b09f49f611de773fb2b--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f3d6b09f49f611de773fb2b>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f3d6b25f49f611de773fb55--gamut-preview.netlify.app>https://5f3d6b25f49f611de773fb55--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f3d6b25f49f611de773fb55>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.1-alpha.f622d3.0<br/>@codecademy/gamut@14.0.1-alpha.f622d3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f3d6b9b715f8a0c903b5d73--gamut-preview.netlify.app>https://5f3d6b9b715f8a0c903b5d73--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f3d6b9b715f8a0c903b5d73>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f3d6bc6715f8a216a3b5ee9--gamut-preview.netlify.app>https://5f3d6bc6715f8a216a3b5ee9--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f3d6bc6715f8a216a3b5ee9>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f3d6e6cc7c32844e13ffcda--gamut-preview.netlify.app>https://5f3d6e6cc7c32844e13ffcda--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f3d6e6cc7c32844e13ffcda>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f3d705fbf676750536de058--gamut-preview.netlify.app>https://5f3d705fbf676750536de058--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f3d705fbf676750536de058>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f3d760a61c71777b9219564--gamut-preview.netlify.app>https://5f3d760a61c71777b9219564--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f3d760a61c71777b9219564>Deploy Logs</a></p>

File: packages/styleguide/stories/Core/Atoms/Badge.stories.mdx [hunk 1]
```diff
@@ -23,10 +21,12 @@ Badges can be used within inline text.
 
 <Canvas>
   <Story name="Inline Badge">
-    <>
-      <Badge>css</Badge>
-      <Badge>css</Badge>
-      <Badge>css</Badge>
-    </>
+    {(args) => (
```
Comment (JoshuaKGoldberg) [inline packages/styleguide/stories/Core/Atoms/Badge.stories.mdx:24]: All these unused `args` variables... can we get rid of them?

Comment (codecaaron) [reply packages/styleguide/stories/Core/Atoms/Badge.stories.mdx:24]: Unfortunately, the AST cannot figure out that it should be run through the formatted without there being that single argument :(

**Pull request merged**